app/src/main/java/org/mifos/mobile/models/client/Client.kt

```diff
@@ -1,132 +1,60 @@
 package org.mifos.mobile.models.client
 
-import android.os.Parcel
 import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
 import org.mifos.mobile.models.Timeline
-import java.util.*
 
 /**
  * @author Vishwajeet
  * @since 20/06/16
  */
-class Client : Parcelable {
-    var id = 0
+@Parcelize
+data class Client(
+    var id: Int = 0,
 
-    var accountNo: String? = null
+    var accountNo: String? = null,
 
-    private var status: Status? = null
+    private var status: Status? = null,
 
-    private var active: Boolean? = null
+    private var active: Boolean? = null,
 
-    var activationDate: List<Int?>? = ArrayList()
+    var activationDate: List<Int?>? = ArrayList(),
```

Comment:
No need for allowing nullable in the list. Also since you are setting the default value as an empty ArrayList, no need for allowing nullable as `activationDate` value